app/views/users/_org_admin.html.erb

```diff
@@ -60,9 +60,10 @@
     </ol>
     <p>
       Your secret code is:
-      <clipboard-copy for="settings-org-secret" onclick="nextElementSibling.hidden = false">
+      <clipboard-copy for="settings-org-secret" onclick="nextElementSibling.hidden = false" data-controller="copy-clipboard">
         <input type="text" class="ff-monospace ff-s fw-medium" id="settings-org-secret" value="<%= @organization.secret %>" readonly aria-label="Organization secret (to be rotated regularly)">
         <%= inline_svg_tag("copy.svg", aria: true, class: "crayons-icon", title: "Copy to clipboard") %>
+        <button data-action="click->copy-clipboard#handleCopyClick">
```

Comment:
Not sure how you want to set up how the button looks, this `<button data-action="click->copy-clipboard#handleCopyClick">` just adds another small button to click and the `inline_svg_tag` is where the clipboard image/button is currently is coming from.